packages/web-app/src/components/executionWidget/index.tsx

```diff
@@ -57,9 +57,12 @@ export const ExecutionWidget: React.FC<ExecutionWidgetProps> = ({
       ) : (
         <>
           <Content>
-            {actions.map((action, index: number) => (
-              <ActionsFilter {...{action}} key={index} type={action.name} />
-            ))}
+            {actions?.map((action, index: number) => {
```

Comment:
For me TS says that `actions` is always an array. I.e., never `null` or `undefined`. Which makes sense sinse we're assigning `[]` by default on line 30. So the question mark is not necessary :)

```suggestion
            {actions.map((action, index: number) => {
```